### PR TITLE
enhance: support templates for expression in Restful api

### PR DESCRIPTION
**Pull request by smellthemoon**
#36672

### CODE REVIEW

File: internal/distributed/proxy/httpserver/utils.go [hunk 1]
```diff
@@ -1679,3 +1680,170 @@ func RequestHandlerFunc(c *gin.Context) {
...
+	}
+	return dtype
+}
+
+func getTemplateType(value interface{}) schemapb.DataType {
+	switch v := value.(type) {
+	case bool:
+		return schemapb.DataType_Bool
+	case string:
+		return schemapb.DataType_String
```
Comment (xiaocai2333) [inline internal/distributed/proxy/httpserver/utils.go:1788]: add int64? If it's an int number like 1, 2, 3, will asserting it as int64 succeed?

**Review by xiaocai2333 — APPROVED** [review]
Add expression params for delete request.

File: internal/distributed/proxy/httpserver/utils.go (hunk 1 above)
Comment (smellthemoon) [reply internal/distributed/proxy/httpserver/utils.go:1788]: no, all the number in json can only be asserted as float64.

**Review by congqixia — APPROVED** [review]
/approve

**Pull request merged**